src/utils/oclif/baseCommand.ts

```diff
@@ -61,7 +61,8 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
     if (err.message.includes('See more help with --help')) {
       this.warn(err.message.replace('--help', `\x1B[93m${process.argv.at(-2)} ${process.argv.at(-1)} -h or --help\x1B[0m`))
     } else {
-      this.warn(err)
+      console.error(err.message)
+      process.exit(1)
```

Comment:
We tried using `this.exit` instead of `process` but it dumped the entire stack trace. What we want is a simple error message to stderr and the return code set correctly. Working within `oclif` doesn't seem to want to allow this.